package.json

```diff
@@ -1,21 +1,25 @@
 {
-    "name": "starfleet",
-    "version": "0.0.1",
-    "license": "MIT",
-    "private": true,
     "devDependencies": {
-        "@symfony/webpack-encore": "^0.17.0",
+        "@symfony/stimulus-bridge": "^1.1.0",
+        "@symfony/webpack-encore": "^0.32.0",
         "babel-core": "^6.26.0",
         "babel-loader": "^7.1.2",
         "babel-preset-env": "^1.6.1",
         "babel-preset-es2015": "^6.24.1",
         "babel-preset-stage-2": "^6.24.1",
         "copy-webpack-plugin": "^4.2.1",
+        "core-js": "^3.0.0",
         "node-sass": "^4.7.2",
         "prettier": "1.8.2",
-        "sass-loader": "^6.0.6",
-        "webpack-notifier": "^1.5.0"
+        "regenerator-runtime": "^0.13.2",
+        "sass-loader": "^9.0.1",
+        "stimulus": "^2.0.0",
+        "webpack-notifier": "^1.6.0"
     },
+    "name": "starfleet",
+    "version": "0.0.1",
+    "license": "MIT",
+    "private": true,
```

Comment:
This should be at the top of the document + Please avoid creating useless diff (code deplacement bring nothing)